force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +486,17 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
+        Integer p1Age = person1.getAge();
+        Integer p2Age = person2.getAge();
+
+        if (p1Age > p2Age) {
+            return person1;
+        } else if (p2Age > p1Age) {
+            return person2;
+        } else {
+            return person1;
+        }
+
+        // Replace null with the variable you used to store the result
     }
```

Comment:
Good work for problems I like. I like to use the technique of replacing a conditional with a guard clause that I mentioned in Slack a while ago. 

If the values are equal you can return either. 

```
  if (person1.getAge() > person2.getAge()) {
            return person1;
        } 
        return person2;
        ```